internal/store/horizontalpodautoscaler.go

```diff
@@ -43,15 +43,15 @@ func (m MetricTargetType) String() string {
 }
 
 var (
-	descHorizontalPodAutoscalerLabelsName          = "kube_hpa_labels"
+	descHorizontalPodAutoscalerLabelsName          = "kube_horizontalpodautoscaler_labels"
 	descHorizontalPodAutoscalerLabelsHelp          = "Kubernetes labels converted to Prometheus labels."
 	descHorizontalPodAutoscalerLabelsDefaultLabels = []string{"namespace", "hpa"}
```

Comment:
```suggestion
	descHorizontalPodAutoscalerLabelsDefaultLabels = []string{"namespace", "horizontalpodautoscaler"}
```

Reply:
Just providing the suggestion, I wouldn't suggest committing this suggestion. Can you change this manually?

Reply:
Done. Can you let met rerun the tests please ?